eng/pipelines/templates/steps/analyze.yml

```diff
@@ -24,6 +24,14 @@ steps:
     failOnStderr: false
     workingDirectory: '${{parameters.GoWorkspace}}'
 
+  - pwsh: |
+      $modDirs = ./eng/scripts/get_module_dirs.ps1 -serviceDir $(SCOPE)
+      foreach ($md in $modDirs) {
+        Get-ChildItem $md/doc.go
```

Comment:
Is the idea that you want this to fail if it isn't present? If so perhaps we should do a Test-Path and give a nicer error message if it is missing.